lib/index.js

```diff
@@ -0,0 +1,15 @@
+/* @flow */
+
+import AtomDebugger from './main'
+
+export default {
+  instance: null,
+  activate() {
+    require('atom-package-deps').install('debugger') // eslint-disable-line
```

Comment:
Well the package is not depending on itself 😉 
It's just telling `atom-package-deps` the name of the *current* package to install dependencies of